serverless.yml

```diff
@@ -370,28 +370,55 @@ functions:
             asset: "digg-wbtc"
       - schedule:
           name: ${self:service}-${self:custom.stage}-index-sushiswap-badger-wbtc
-          description: "index historical data for badger sushiswap badger-wbtc jar"
+          description: "index historical data for badger sushiswap badger-wbtc vault"
           rate: rate(5 minutes)
           input:
             contract: "0x1862a18181346ebd9edaf800804f89190def24a5"
             createdBlock: 11537600
             asset: "slp-badger-wbtc"
       - schedule:
           name: ${self:service}-${self:custom.stage}-index-sushiswap-wbtc-eth
-          description: "index historical data for badger sushiswap wbtc-eth jar"
+          description: "index historical data for badger sushiswap wbtc-eth vault"
           rate: rate(5 minutes)
           input:
             contract: "0x758a43ee2bff8230eeb784879cdcff4828f2544d"
             createdBlock: 11539529
             asset: "slp-wbtc-eth"
       - schedule:
           name: ${self:service}-${self:custom.stage}-index-sushiswap-digg-wbtc
-          description: "index historical data for badger sushiswap digg-wbtc jar"
+          description: "index historical data for badger sushiswap digg-wbtc vault"
           rate: rate(5 minutes)
           input:
             contract: "0x88128580acdd9c04ce47afce196875747bf2a9f6"
             createdBlock: 11681030
             asset: "slp-digg-wbtc"
+      # - schedule:
```

Comment:
these are indexing at wrong intervals, so we will remove them for now. we will need to delete all these entries and re enable these once the fixes are in